.eslintignore

```diff
@@ -1,7 +1,5 @@
 dist
+build
```

Comment:
Does build exist or do we only use dist folder?

Reply:
god poeng! jeg bare hadde det lokalt. skal teste.